pkg/authorizer/interfaces.go

```diff
@@ -18,3 +18,13 @@ type Interface interface {
 type ACRTokenExchanger interface {
 	ExchangeACRAccessToken(armToken types.AccessToken, tenantID, acrFQDN string) (types.AccessToken, error)
 }
+
+// MIARMTokenRetriever is the interface to retrieve an ARM access token via managed identity.
```

Comment:
Nit: comment says `MIARMtokenRetriever` instead of `ManagedIdentityARMTokenRetriieever`

Reply:
Resolved in the most recent commit. Please check it out. Thanks!